lib/Service.php

```diff
@@ -189,6 +189,27 @@ function write($rootElementName, $value, $contextUri = null) {
 
     }
 
+    /**
+     * Maps the given plain-old-php-object (popo) to the given rootElementName within the given namespace.
+     *
+     * The $rootElement must be specified in clark notation.
+     *
+     * @param string $rootElementName
+     * @param object $popo
+     * @param string $namespace
+     */
+    function mapPopo($rootElementName, $popo, $namespace) {
+        if (!is_object($popo)) {
```

Comment:
Maybe we can do both? I feel it's more natural to allow just classes and force people to create their VO's without constructors, but I can definitely see an advantage to both